File: backend/order-service/src/main/java/com/quisin/order/service/client/MenuItemResponse.java
package com.quisin.order.service.client;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class MenuItemResponse {
    private Long id;
    private Long restaurantId;
    private String name;
    private String description;
    private BigDecimal price;
    private Boolean available;
    private String category;
    private String imageUrl;
} 

File: backend/menu-service/src/main/kotlin/com/quisin/menu/dto/MenuItemResponse.kt
package com.quisin.menu.dto

import com.quisin.menu.domain.MenuItemCategory
import com.quisin.menu.domain.MenuItemStatus
import java.math.BigDecimal
import java.time.LocalDateTime
import java.util.UUID

data class MenuItemResponse(
    val id: UUID,
    val restaurantId: UUID,
    val name: String,
    val description: String?,
    val price: BigDecimal,
    val category: MenuItemCategory,
    val imageUrl: String?,
    val currentStock: Int,
    val maxStock: Int,
    val status: MenuItemStatus,
    val preparationTime: Int?,
    val calories: Int?,
    val allergens: List<String>,
    val tags: List<String>,
    val createdAt: LocalDateTime,
    val updatedAt: LocalDateTime
) 